plugins/source/googleads/client/context.go

```diff
@@ -13,21 +13,13 @@ const (
 
 func (c *Client) OutgoingContext(ctx context.Context) context.Context {
 	ctx = c.addDeveloperToken(ctx)
-	return c.addLoginCustomerID(ctx)
+	return addLoginCustomerID(ctx, c.ManagerID)
```

Comment:
Forgot to add `if len(c.ManagerID) > 0 { ...`

Reply:
I took a look at the logic and the manager ID is always present, so omitted intentionally.

Reply:
But I'll add it to be consistent